src/memote/checks/Consistency.jl

```diff
@@ -129,4 +129,56 @@ function model_is_consistent(
     termination_status(opt_model) == OPTIMAL
 end
 
+"""
+$(TYPEDSIGNATURES)
+
+Returns a list of all metabolites that aren't part of any reactions.
+"""
+function find_disconnected_metabolites(model::MetabolicModel)
+    mids = deepcopy(metabolites(model))
+    for rid in reactions(model)
+        rid_m = reaction_stoichiometry(model, rid)
+        for mid in mids
+            haskey(rid_m, mid) && deleteat!(mids, findall(x->x == mid, mids))
+        end
+    end
+    return mids
+end
+
+"""
+$(TYPEDSIGNATURES)
+
+A function to find reactions able to carry unlimited flux under default conditions.
+The function compares the fluxes of the model calculated by FVA (using flux_variability_analysis_dict)
+with the median bounds of the model (+/- treshold) and returns the fluxes which are grearter or smaller than 
+the bounds in a two seperate dictionaries.
+"""
+function unbounded_flux_in_default_medium(model::MetabolicModel, fva_result::Tuple{Any,Any}, config = memote_config)
+    tol = config.consistency.tolerance_threshold
+
+    if isempty(fva_result)
+        throw(ArgumentError("fva_result is empty"))
```

Comment:
change the error once fva is called internally

Reply:
re my previous comment, this can be something like
```julia
all(!isnothing, fva_result) || throw(ArgumentError("fva_result is borked"))
```
but you'll likely need a slightly more advanced condition there :D

Reply:
What exactly do you mean by slightly more advanced? We don't have a lot of experience with FVA so we are unsure what to check beyond if the result is empty.

Reply:
individual fields of the FVA result may be `nothing` in case the solver somehow fails to optimize it in that direction. Which is unlikely in an optimal world, but very probable with imprecise floats that we have :D

For now this `isempty` is OK I guess.